src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferApiClientUnitTest.java

```diff
@@ -170,9 +169,8 @@ public void testDownload_GenericException() {
         when(restTemplate.execute(eq(DOWNLOAD_URI), eq(HttpMethod.GET), any(RequestCallback.class), ArgumentMatchers.<ResponseExtractor<ClientHttpResponse>>any(), any(FileTransferApiClientResponse.class)))
             .thenThrow(exception);
 
-        RestClientException thrown = assertThrows(RestClientException.class, () ->
+        assertThrows(RestClientException.class, () ->
             fileTransferApiClient.download(FILE_ID, servletResponse));
-        assertEquals(thrown.getMessage(), thrown.getMessage());
     }
```

Comment:
the assert for exception message is necessary. if you look at the original, we were checking that the exception message thrown by the RestTemplate is passed back in the exception from the fileTransferApiClient.
we should check like
assertEquals(EXCEPTION_MESSAGE, thrown.getMessage());